index.html

```diff
@@ -2,20 +2,55 @@
 <html>
   <head>
     <title>Todo app</title>
+
   </head>
   <body>
-    <h1>My Todo app</h1>
-    <form>
-    	<input type="Text" name="EnterTask" value="EnterTask">
-    	
-    	<input type="button" name="Submit" id="Submit" value="submit">
+  <div id="myDIV" class="header">
+  <h2>My To Do List</h2>
+  <input type="text" id="myInput" placeholder="Title..."> <br>
+
+  <input type="button" name="Add New Task" value="Add New Task" span onclick="newElement()" class="addBtn">
```

Comment:
this should be self closing tag

Reply:
no need to use `span`

Reply:
we should use`type="submit"` so that on press of enter also a new task can be added.
You can use `event.preventDefault()` to stop refresh on submit

Reply:
> this should be self closing tag

which tag?